src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -670,17 +670,11 @@ def process_declarations(self, parent, nodes, arg_list):
 
                 if initialisation:
                     if has_constant_value:
-                        # If it is a parameter, get the initialization value
+                        # If it is a parameter parse its initialization
+                        tmp = Node()
                         expr = initialisation.items[1]
-                        if isinstance(expr, Fortran2003.NumberBase):
-                            value_str = expr.items[0]
-                            # Convert string literal to the Symbol datatype
-                            ct_value = TYPE_MAP_TO_PYTHON[datatype](value_str)
```

Comment:
pylint now says that TYPE_MAP_TO_PYTHON is unused in this module.

Reply:
Deleted